awscli/customizations/s3/results.py

```diff
@@ -239,35 +246,40 @@ class ResultPrinter(BaseResultHandler):
         '{transfer_type} failed: {src} to {dest} {exception}'
     )
     WARNING_FORMAT = (
-        'warning: {message}'
+        '{message}'
```

Comment:
Would it make sense instead to move the `warning: ` and `error: ` into these format strings?